doc/release-notes-15954.md

```diff
@@ -0,0 +1,4 @@
+Configuration option changes
+-----------------------------
+
+Importing blocks upon startup via the `bootstrap.dat` file no longer occurs by default. The file must now be specified with `-loadblock=<file>`.
```

Comment:
@UdjinM6 are you okay with this change?

Reply:
yeah, why not

Reply:
It's a bit awkward but that's how it was merged in 17044 (they forgot to rename `doc/release-notes-15954.md`?)

Reply:
Ah, good point 👍 Done https://github.com/UdjinM6/dash-bootstrap/commit/515606234e6d4599c8e38fb82c0244a2983724f9 :)